src/marqo/core/constants.py

```diff
@@ -14,6 +14,7 @@
 
 MARQO_STRUCTURED_HYBRID_SEARCH_MINIMUM_VERSION = '2.10.0'
 MARQO_UNSTRUCTURED_HYBRID_SEARCH_MINIMUM_VERSION = '2.11.0'
+MARQO_CUSTOM_VECTOR_NORMALIZATION_MINIMUM_VERSION = '2.12.0'
```

Comment:
This should be 2.13

Reply:
Ah, I thought we were releasing this fix in versions post 2.12.0, let me know if not and I can change this

Reply:
Changed as per offline discussion